src/EnergyPlus/CMakeLists.txt

```diff
@@ -570,7 +570,12 @@ install( TARGETS energyplusapi
   LIBRARY DESTINATION ./ 
 )
 
-if (CMAKE_COMPILER_IS_GNUCXX AND ENABLE_LTO)
+if (MINGW)
```

Comment:
@kbenne I had the issues you have been having in mind when I decided to go with this version and rule out LTO for MinGW. Would you like me to try building with LTO to see what happens?